src/mavsdk/core/mavlink_command_sender.cpp

```diff
@@ -131,7 +131,16 @@ void MavlinkCommandSender::receive_command_ack(mavlink_message_t message)
     mavlink_command_ack_t command_ack;
     mavlink_msg_command_ack_decode(&message, &command_ack);
 
-    if ((command_ack.target_system &&
+    // Accept all broadcast command ack's
+    if(command_ack.target_component == MAV_COMP_ID_ALL)
```

Comment:
I think I can see why this one is required. Otherwise we filter out component 0.